File: market-domain/src/main/java/com/example/market/domain/settlement/PayoutStatus.java
package com.example.market.domain.settlement;

public enum PayoutStatus {
    SCHEDULED,   // 정산 예정
    SENT,        // 송금 요청됨
    COMPLETED,   // 정산 완료
    FAILED;      // 송금 실패
    public boolean isTerminal() { return this == COMPLETED || this == FAILED; }
}

File: market-domain/src/main/kotlin/com/example/market/domain/settlement/PayoutStatus.kt
package com.example.market.domain.settlement

enum class PayoutStatus {
    /** 정산 예정 */
    SCHEDULED,

    /** 송금 요청됨 */
    SENT,

    /** 정산 완료 */
    COMPLETED,

    /** 송금 실패 */
    FAILED;

    fun isTerminal(): Boolean = this == COMPLETED || this == FAILED
}
